### ISSUE TITLE
Java User Groups

### ISSUE THREAD

**Issue by McPringle** [marchof/javabubble#70]
Hi! What do you think about adding a section for Java User Groups? Do you count them to be a part of the Java Bubble?

**Comment by marchof** [comment]
Absolutely! We already have the Java Champions account. What about flagging entries e.g. as `individual` or `organisation`?

**Comment by marchof** [comment]
Looking through incoming entries there is probably another category `product` (e.g. GraalVM)

**Comment by McPringle** [comment]
The result should be different CSV files. So people can decide to import only individuals, JUGs, or organizations.

**Comment by boyarsky** [comment]
I think three categories are useful. I think **userGroup** is a better name than organization. Because some companies and non profits call themselves organization.

I also like your idea of **product** for things like graal or azul. This could get spammy but PRs should serve as a good gate. I think we all know intuitively who is part of our community!

**Comment by marchof** [comment]
Re `product`: Many FOSS projects have their own social media handles, that was my original intention

Re `usergroup`: Fine to me, but then we still need a category e.g. for "Java Champions". Maybe just `group`?

**Comment by Bukama** [comment]
There is already a JUG overview, you could also argue that's one tracked by Oracle

https://dev.java/community/jugs/

**Comment by McPringle** [comment]
> There is already a JUG overview, you could also argue that's one tracked by Oracle

But it does not contain any social media information at all.
